libraries/pyopaeuio/CMakeLists.txt

```diff
@@ -24,11 +24,19 @@
 ## ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,  EVEN IF ADVISED OF THE
 ## POSSIBILITY OF SUCH DAMAGE.
 
+set(PYOPAE_PYBIND11_VERSION "2.2.4")
```

Comment:
Can we remove this definition of PYOPAE_PYBIND11_VERSION and the one from pyopae, and just use the global one defined here? https://github.com/OFS/opae-sdk/blob/master/CMakeLists.txt#L425